jib-maven-plugin/README.md

```diff
@@ -309,6 +309,8 @@ Property | Type | Default | Description
 --- | --- | --- | ---
 `from` | file | `[(project-dir)/src/main/jib]` | The source directory. Can be absolute or relative to the project root.
 `into` | string | `/` | The absolute unix path on the container to copy the extra directory contents into.
+`includes` | list | *None* | Glob patterns for including files. See [Adding Arbitrary Files to the Image](#adding-arbitrary-files-to-the-image) for an example.
```

Comment:
Do we want to document that glob pattern matching here behaves differently from maven-resources-plugin?

Reply:
I think it's okay without it. People will follow the link to see the example, where I added the correct usage to cover the case we discussed.